plugin/db/mongodb/mongodb.go

```diff
@@ -0,0 +1,139 @@
+// Package mongodb is the plugin for MongoDB driver.
+package mongodb
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"io"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+
+	"github.com/pkg/errors"
+
+	"github.com/bytebase/bytebase/plugin/db"
+)
+
+var (
+	_ db.Driver = (*Driver)(nil)
+)
+
+func init() {
+	db.Register(db.MongoDB, newDriver)
+}
+
+// Driver is the MongoDB driver.
+type Driver struct {
+	client *mongo.Client
+}
+
+func newDriver(_ db.DriverConfig) db.Driver {
+	return &Driver{}
+}
+
+// Open opens a MongoDB driver.
+func (driver *Driver) Open(ctx context.Context, _ db.Type, connCfg db.ConnectionConfig, _ db.ConnectionContext) (db.Driver, error) {
+	connectionURI := getMongoDBConnectionURI(connCfg)
+	client, err := mongo.NewClient(options.Client().ApplyURI(connectionURI))
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create MongoDB client")
+	}
+	err = client.Connect(ctx)
```

Comment:
Is this Connection() required? Could we have a later step connecting to the server lazily?

Reply:
MongoDB doesn't support connecting lazily, we can implement this by calling` tryConnect(){ if(notconnect){connnect} }` later.